16# postpone a task

DraganKec opened the pull request.

Commit:
#16 Create getTasks method in TasksService
#16 Add postpone task in controller

Commit:
#16 Add commented request and return

cctraining reviewed (commented):
Missing test for postponeTasks method.

nikola-lisac commented:
- Move the logic to the service class
- Use Integer instead of int
- Rename method in service to getTask because it returns a single task
- Delete commented example code in the Controller and post it as a comment here on the pull request

Commit:
#16 Move logic from controller to service
#16 Rename method from getTasks to GetTask

Commit:
#16 Change java Date to java 8 LocalDate
#16 Add LocalDate deserializer and serializer class

DraganKec commented:
POST
http://localhost:8080/tasks/6

Return
{
 "id": 6,
 "name": "task",
 "createdAt": "2017-08-29",
 "completed": false
}

BojanJakic commented on src/main/java/ba/codecentric/prioritytasks/service/TaskService.java:
```diff
@@ -23,10 +23,25 @@ public Task saveTask(Task task) {
+        List<Task> tttt = taskRepository.findAll();
         return taskRepository.findAll();
     }
 
```
Do you need tttt ?

Commit:
#16 Remove unnecessary variable
#16 Change name of method from getTask to postponeTasks
#16 Add request for get one task

Commit:
Merge branch 'master' into 16#_Postpone_a_task

# Conflicts:
#	src/main/java/ba/codecentric/prioritytasks/controller/TaskController.java
#	src/main/java/ba/codecentric/prioritytasks/repository/TaskRepository.java
#	src/main/java/ba/codecentric/prioritytasks/service/TaskService.java

Commit:
#16 Merge with master
